charts/seed-controlplane/charts/kube-apiserver/templates/deployment.yaml

```diff
@@ -44,6 +44,20 @@ spec:
         networking.gardener.cloud/to-shoot-networks: allowed
         networking.gardener.cloud/from-prometheus: allowed
     spec:
+      affinity:
+        podAntiAffinity:
+          requiredDuringSchedulingIgnoredDuringExecution:
```

Comment:
Do we really need to `require` it or can we also `prefer` it? In most cases `prefer` should yield the same result, except for small/bin-packed clusters and then it would block, which is something that may become dangerous to the stability of the control plane. Even if 2 pods end up on the same node, normal eviction would work even with the PDB (one by one within the node drain timeout) and only if the node dies, multiple API servers would be gone, but that's highly unlikely and even if, API servers start fast and if there are more than the ones on that particular node, then the others might be able to support the load for a brief time window. Bottom line, I would go with `prefer` as it's closer to what we had before/less risky. WDYT?

Reply:
Sure, makes sense, thanks for spotting.